POS/Integration/05_ConfigureGlobalSettings.md

```diff
@@ -132,26 +124,32 @@ A payment method has been created in the *Order management* module, see [Create
 
 4. Enter a name for the payment method in POS in the *Name* field.   
 
-    > [Info] The name of the payment method in POS can differ from the name of the payment method in the *Order management* module. It is only necessary that the ID of the payment method in *Venduo POS* and *Order management* match.
+    > [Info] The name of the payment method in POS can differ from the name of the payment method in the *Order Management* module. It is only necessary that the ID (key) of the payment method in *Venduo POS* and *Order Management* match.
```

Comment:
Genau das meinte ich oben. OK, Titel/Name muss nicht stimmen, es ist aber grundsätzlich das gleiche gemeint.
Vielleicht deutlicher das erklären: The Key in POS must match the ID in OM... Diese Aussage finde ich etwas vague/verwirrend.